probe-rs/src/architecture/riscv/communication_interface.rs

```diff
@@ -89,15 +89,28 @@ impl AbstractCommandErrorKind {
 /// register.
 #[allow(dead_code)]
 #[derive(Debug, Copy, Clone, PartialEq)]
-enum DebugModuleVersion {
+pub enum DebugModuleVersion {
     /// There is no debug module present.
-    NoModule = 0,
+    NoModule,
     /// The debug module confirms to the version 0.11 of the RISCV Debug Specification.
```

Comment:
I think it should be "conforms" in both comments here. Not part of the change but since there is things changed here, you might wanna fix this too :)